docs/guides/js-waku/reliable-channels.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Send and Receive Messages in a Reliable Channel
+hide_table_of_contents: true
+---
+
+Learn how to send and receive messages with a convenient SDK that provide various reliable functionalities out-of-the-box.
+
+:::warning
+This is an experimental feature and has a number of [limitations](https://github.com/waku-org/js-waku/pull/2526).
+:::
+
+## Import Waku SDK
+
+```shell
+npm install @waku/sdk@0.0.35-4d5c152.0
```

Comment:
it's ok to mention `@waku/sdk@0.0.35` as we will have it released in a week or two

Reply:
```suggestion
npm install @waku/sdk@0.0.35
```